examples/index.md

```diff
@@ -126,3 +126,119 @@ Content-Type: application/vnd.api+json
 for readability. In practice, these characters must be percent-encoded, as
 noted in the base specification.
 
+## Error Objects
+
+Examples of how [example objects](http://jsonapi.org/format/#error-objects) work.
+
+### Responses to failed resource creation:
+
+Response with one error object:
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "source": { "pointer": "data/attributes/first_name" },
+      "detail": "is too short"
+    }
+  ]
+}
+```
+
+Response with status:
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "source": { "pointer": "data/attributes/first_name" },
+      "detail": "is too short"
+    },
+    {
+      "status": "422",
```

Comment:
Having `status` in isolation seems a bit odd. In this example, I'd expect it to be part of the first error object.

I think an example that shows multiple different errors with different `status` values would be great. For example, one error object could be a `404` and another `422`, and the status for the whole response could be `400`.